sql/functions.go

```diff
@@ -0,0 +1,16 @@
+package sql
+
+import (
+	"fmt"
+
+	"crawshaw.io/sqlite"
+)
+
+func registerFunctions(conn *sqlite.Conn) error {
```

Comment:
can you comment on/explain what this function is for and why is it needed? i don't really understand

Reply:
sqlite doesn't have native support for uint64, so you can't add them or sort by uint64

Reply:
can you add comment that explain this. it's not obvious for readers.